spectree/spec.py

```diff
@@ -15,10 +17,22 @@ class SpecTree:
     :param str backend_name: choose from ('flask', 'falcon', 'starlette')
     :param backend: a backend that inherit `SpecTree.plugins.base.BasePlugin`
     :param app: backend framework application instance (you can also register to it later)
+    :param before: a callback function of the form :meth:`spectree.utils.default_before_handler`
+        ``func(req, resp, req_validation_error, instance)``
+        that will be called after the request validation before the endpoint function
+    :param after: a callback function of the form :meth:`spectree.utils.default_after_handler`
+        ``func(req, resp, resp_validation_error, instance)``
+        that will be called after the response validation
     :param kwargs: update default :class:`spectree.config.Config`
     """
 
-    def __init__(self, backend_name='base', backend=None, app=None, **kwargs):
+    def __init__(self,
+                 backend_name='base', backend=None,
+                 app=None,
+                 before=default_before_handler, after=default_after_handler,
+                 **kwargs):
+        self.before = before
```

Comment:
Is it possible to move this to the validate function instead as an argument. This would allow for different callback function for each decorators.

Reply:
Good point, I'll support both.